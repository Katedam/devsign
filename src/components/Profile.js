import React from 'react';
import CurrentUser from '../containers/CurrentUser';
import CreateChirpForm from './CreateChirpForm';
import Nav from './Nav';
import { Main, Section } from '../styles/mainStyle';
import UserProfileChirps from '../containers/UserProfileChirps';


function Profile() {
  return (
    <>
    <Nav />
      <CreateChirpForm />
    <Main>
      <CurrentUser />
      <Section>
        <UserProfileChirps />
      </Section>
    </Main>
    </>
  );
}

export default Profile;
