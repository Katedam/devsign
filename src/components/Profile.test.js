import React from 'react';
import Profile from './Profile';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

describe('Profile component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <MemoryRouter>
        <Profile />
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
