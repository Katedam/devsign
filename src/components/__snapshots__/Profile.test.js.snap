// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile component matches a snapshot 1`] = `
Array [
  <nav
    className="sc-hMqMXs dbUHYG"
  >
    <ul
      className="sc-kEYyzF kbZRkL"
    >
      <a
        className="sc-bdVaJa cVFclJ"
        href="/home"
        onClick={[Function]}
      >
        <li
          className="sc-gZMcBi sc-iAyFgw uYoYC"
        >
          Chirp
        </li>
      </a>
      <a
        className="sc-bdVaJa cVFclJ"
        href="/home"
        onClick={[Function]}
      >
        <li
          className="sc-gZMcBi sc-kkGfuU fhFPuL"
        >
          Home
        </li>
      </a>
      <a
        className="sc-bdVaJa cVFclJ"
        href="/profile"
        onClick={[Function]}
      >
        <li
          className="sc-gZMcBi sc-kkGfuU fhFPuL"
        >
          Profile
        </li>
      </a>
    </ul>
    <a
      className="sc-bdVaJa cVFclJ"
      href="/profile"
      onClick={[Function]}
    >
      <img
        className="sc-hSdWYo hcsGuV"
        src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
      />
    </a>
    <form
      className="sc-ifAKCX sc-dxgOiQ ZDpZY"
    >
      <label
        className="sc-eNQAEJ iqowgK"
        htmlFor="search"
      >
        Search
      </label>
      <input
        className="sc-bZQynM sc-ckVGcZ dfGCWT"
        id="search"
        readOnly={true}
        type="text"
        value=""
      />
      <label
        className="sc-eNQAEJ iqowgK"
        htmlFor="searchButton"
      />
      <button
        className="sc-gzVnrw sc-jKJlTe hHTIcO"
        id="searchButton"
      >
        Search
      </button>
    </form>
  </nav>,
  <section>
    <form
      className="sc-ifAKCX sc-kgoBCf fdofMG"
    >
      <label
        className="sc-EHOje kYkHpN"
        htmlFor="newChirpText"
      >
        Name
      </label>
      <input
        className="sc-bZQynM sc-kGXeez kJKpum"
        id="newChirpText"
        name="newChirpText"
        readOnly={true}
        type="text"
      />
      <label
        className="sc-EHOje kYkHpN"
        htmlFor="newChirp"
      />
      <button
        className="sc-gzVnrw sc-kpOJdX gFAKbC"
        id="newChirp"
      >
        Chirp
      </button>
    </form>
  </section>,
  <main
    className="sc-bxivhb jHTRIX"
  >
    <aside
      className="sc-chPdSV ecAaUG"
    >
      <img
        className="sc-gqjmRU hTssrt"
        name="profileImage"
        src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
      />
      <h2>
        Kaiya
         
        <span
          className="sc-jTzLTM bxQZkh"
        >
          @ladybeard
        </span>
      </h2>
      <p
        className="sc-fjdhpX eUAEJX"
      >
        Fullstack MERN Developer / Animal Lover / Hiker
      </p>
      <p
        className="sc-fjdhpX eUAEJX"
      >
        Portland, OR
      </p>
    </aside>
    <section
      className="sc-cSHVUG jxaPmy"
    >
      <section>
        <ul
          className="sc-iwsKbI jzeEWW"
        >
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kAzzGY bSaNbd"
            >
              <header>
                <img
                  className="sc-gqjmRU hTssrt"
                  src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-jTzLTM bxQZkh"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-jzJRlG fefLBr"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
        </ul>
      </section>
    </section>
  </main>,
]
`;
